dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -279,6 +376,11 @@ def state_name
       sanitize_form_data_hash[:state]
     end
 
+    def first_name
```

Comment:
I'm tempted to say this should be in application base but that's excessively DRY

Reply:
Is it in the principal approval form as well? We don't (currently) have this method in the Facilitator Application. If we end up using it in more places, we can move it to the base later.